viz.pl

```diff
@@ -20,32 +13,21 @@
     graph(Name, Nodes, Edges, SubGraphs),
     export(graph(Name, Nodes, Edges, SubGraphs)).
 
-export :-
-    example_(Graph),
-    export(Graph).
-
 export(Graph) :-
     gv_export(
-        'graph.svg',
+        'graph.png',
```

Comment:
svg is the superior format but theres prob a reason for this.

Reply:
Kills Reg's computer

Reply:
For some reason my computer doesn't comprehend svgs